sensing/autoware_pointcloud_preprocessor/schema/filter.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Filter Class",
+  "type": "object",
+  "definitions": {
+    "filter": {
+      "type": "object",
+      "properties": {
+        "input_frame": {
+          "type": "string",
+          "description": "input frame id",
```

Comment:
Since we are doing checks, I think it would be nice if we could detect empty frames (empty frames cause errors in ROS)

https://json-schema.org/understanding-json-schema/reference/string

Reply:
Understood. 
However, the design of the filter class allows the empty string if the user doesn't want to change the input frame and the output frame. Therefore, for this PR, I couldn't set the restriction.

But this can be handled if the filter class is rewritten.

Reply:
:+1: